pennylane_lightning/src/tests/Test_AVXSingleQubitGateHelpers.cpp

```diff
@@ -0,0 +1,344 @@
+#include "cpu_kernels/avx_common/SingleQubitGateHelper.hpp"
```

Comment:
Could we add `cpu_kernels` and `avx_common` to the build?

Reply:
I think we can keep the current structure at the moment. They are only used inside the `gate` submodule.